packages/kolibri-components/src/KThemePlugin.js

```diff
@@ -41,6 +41,9 @@ export default function KThemePlugin(Vue) {
           outline: 'none',
         };
       },
+      $inputModality() {
+        return globalThemeState.inputModality;
+      },
```

Comment:
great

Reply:
I had tried to add it as a computed prop solely to the media player component, but it wouldn't update. Although, this works even though it's nearly the same.